### PR TITLE
Add docs about managing agent tags

### LINKED ISSUE #1978: [Request] Add Fleet documentation about managing agent tags
## Description

In 8.4 we are introducing new features to manage tags: https://github.com/elastic/ingest-dev/issues/1043

There is existing documentation on filtering on tags and creating tags at agent enroll time: https://www.elastic.co/guide/en/fleet/8.3/filter-agent-list-by-tags.html

The documentation should be updated to include the new features to manage tags:
- New action from Agent list to `Add / remove tags` on single and bulk selection of agents https://github.com/elastic/kibana/pull/135320 https://github.com/elastic/kibana/pull/135646
- Ability to Rename / Delete tags from the list of tags https://github.com/elastic/kibana/pull/135712

## Collaboration
- The docs team will lead producing the content

**Contact Person:** @juliaElastic 

## Suggested Target Release

8.4

## Stakeholders

@nimarezainia @jen-huang

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #1978 

(Apologies in advance for any typos. My spell checker stopped working and won't reset.)

### CODE REVIEW

**Comment by jen-huang** [comment]
@juliaElastic, could you review?

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 1]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
...
 image::images/agent-tags.png[Agents list filtered to show agents with the staging tag]
 
-If you haven't applied tags to any {agent}s yet, the list will be empty.
+If you haven't added tags to any {agent}s yet, the list will be empty.
+
+[discrete]
+[[add-tags-in-fleet]]
+== Add, remove, or rename tags in {fleet}
+
+You can use {fleet} to add, remove, or rename tags applied to one or more
```
Comment (juliaElastic) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:25]: we could rephrase this to "add, remove, rename or delete tags"

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 2]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
...
+[options,header]
+|===
+|To... | Do this...
+
+|Create a new tag
+|Type the tag name and click **Create new tag...**. Notice the tag name has
+a check mark to show that the tag has been added to the selected agents.
+
+|Rename a tag
+|Hover over the tag name and click the ellipses button. Enter a new name.
```
Comment (juliaElastic) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:55]: we could be more explicit here, something like `Enter a new name, press Enter or click outside.`

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 3]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
...
+The tag will be renamed in all agents that use it, even agents that are not
+selected.
+
+|Delete a tag
+|Hover over the tag name and click the ellipses button. Click **Delete tag**.
```
Comment (juliaElastic) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:60]: typo: ellipses -> ellipsis

**Review by juliaElastic — APPROVED** [review]
Look good, added some small suggestions.

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 4]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
 [role="screenshot"]
 image::images/agent-tags.png[Agents list filtered to show agents with the staging tag]
 
-If you haven't applied tags to any {agent}s yet, the list will be empty.
+If you haven't added tags to any {agent}s yet, the list will be empty.
+
+[discrete]
+[[add-tags-in-fleet]]
+== Add, remove, or rename tags in {fleet}
```
Comment (dedemorton) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:23]:
```suggestion
== Add, remove, rename, or delete tags in {fleet}
```

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc (hunk 2 above)
Comment (dedemorton) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:55]:
```suggestion
|Hover over the tag name and click the ellipses button. Type a new name and press Enter.
```

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc (hunk 3 above)
Comment (dedemorton) [reply docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:60]: Good catch!

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 5]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
...
+
+. On the **Agents** tab, select one or more agents.
+
+. From the **Actions** menu, click **Add / remove tags**.
++
+[role="screenshot"]
+image::images/add-remove-tags.png[Screenshot of add / remove tags menu]
++
+TIP: Make sure you use the correct **Actions** menu. To manage tags for a single
+agent, click the ellipses button under the **Actions** column. To manage tags
```
Comment (dedemorton) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:41]:
```suggestion
agent, click the ellipsis button under the **Actions** column. To manage tags
```

File: docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc [hunk 6]
```diff
@@ -17,15 +16,64 @@ field for autocompletion.
...
+[options,header]
+|===
+|To... | Do this...
+
+|Create a new tag
+|Type the tag name and click **Create new tag...**. Notice the tag name has
+a check mark to show that the tag has been added to the selected agents.
+
+|Rename a tag
+|Hover over the tag name and click the ellipses button. Type a new name and press Enter.
```
Comment (dedemorton) [inline docs/en/ingest-management/fleet/filter-agent-list-by-tags.asciidoc:55]:
```suggestion
|Hover over the tag name and click the ellipsis button. Type a new name and press Enter.
```
